hip/static/js/index.js

```diff
@@ -23,6 +23,7 @@ import Header from "../../../apps/hip/static/js/header";
 import SideBar from "../../../apps/hip/static/js/sidebar";
 import RightScrollNav from "../../../apps/hip/static/js/rightScrollNav";
 import MobileSearchModal from "../../../apps/hip/static/js/mobileSearchModal";
+import loggedInDropdown from "../../../apps/hip/static/js/loggedInDropdown";
```

Comment:
Any reason this is not uppercased like the other components?

Reply:
No, just zoned out and made it camel case.